doc/nrf/../testing_dfu: nRF54l15: note on pure-DTS build (#16241)
nrfconnect/sdk-nrf: doc/nrf/device_guides/working_with_nrf/nrf54l/testing_dfu.rst

> To compile the SMP server sample for testing secondary image slots on external SPI NOR flash, run the following command:

Reviewer: ```suggestion
To compile the SMP server sample for testing secondary image slots on external SPI NOR flash, run the command based on your partitioning method.
```

.. tabs::

>    .. group-tab:: PM partitioning

Reviewer: ```suggestion
   .. group-tab:: Partition Manager
```

>       To build along with Partition manager:

Reviewer: ```suggestion
      To build with the Partition Manager, run the following command:
```

      .. code-block:: console

         west build -b nrf54l15pdk/nrf54l15/cpuapp -d build/smp_svr_54l zephyr/samples/subsys/mgmt/mcumgr/smp_svr -T sample.mcumgr.smp_svr.bt.nrf54l15pdk.ext_flash

   .. group-tab:: DTS partitioning

>       To build along with DTS partitioning:

Reviewer: ```suggestion
      To build with the DTS partitioning, run the following command:
```
